### DISCUSSION TITLE
support for eslint import/no-unused-modules

### DISCUSSION THREAD

**Discussion by wouter-leistra** [biomejs/biome#4645] [Ideas]
We are looking into switching to Biome from our prettier, eslint setup after several failed attempts to get eslint 9 going. During our initial research we noticed that Biome linter doesn't have any documented support for eslints import/no-unused-modules. (https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-unused-modules.md)

Is this something that is still in the pipeline for Biome to support or has it been chosen not to be pursued.

**Comment by ematipico** [comment]
Please use https://github.com/biomejs/biome/discussions/3 

It's a discussion we pinned to collect proposals like yours. Make sure that your proposal hasn't already been proposed by someone else. Thank you,
